### ISSUE TITLE
Cron monitor check-ins sent with Oban integration are marked as

### ISSUE THREAD

**Issue by rodolfoBee** [getsentry/sentry-elixir#756]
### Environment
SDK version 10.6.0.

### Steps to Reproduce
SDK configuration:

```
config :sentry,
dsn: System.get_env("SENTRY_DSN"),
environment_name: environment,
enable_source_code_context: true,
root_source_code_paths: [File.cwd!()],
integrations: [
oban: [
cron: [enabled: true]
]
```

Jobs are configured using crontab, for example:
```
{"1-59/5 * * * *", Cron.DeleteTeam},
{"2-59/5 * * * *", Cron.GenerateFees},
```

It was tried to manually create the monitor following the [auto-config in the oban integration code](https://github.com/getsentry/sentry-elixir/blob/master/lib/sentry/integrations/oban/cron.ex#L75-L76) with no success.

### Expected Result
Check-ins are sent and accepted by Sentry following the job's crontab

### Actual Result
Check-ins are marked as "Monitor rate limit": 

![Image](https://github.com/user-attachments/assets/3b3a22a0-de79-40cb-8566-e6c4f07abd83)

All monitors are active in Sentry and there is available quota.

**Comment by whatyouhide** [comment]
I’m not sure what Sentry means by "Dropped (Server)". It doesn't give you any details on why the monitor was dropped?

**Comment by rodolfoBee** [comment]
The only reason given is "Monitor Rate Limit", note no check-in is accepted so usual rate limits (6 checkins per monitor per minute) do not apply here. 
How exactly is the check-in envelope created by the oban integration and sent by the SDK?

**Comment by whatyouhide** [comment]
We create the envelope with something along these lines:

```elixir
[
  ~s({"event_id":"#{event_id}"}\n),
  ~s({"type": "check_in", "length": #{byte_size(encoded_check_in)}}\n),
  encoded_check_in,
  ?\n
]
```

You can see this code [here](https://github.com/getsentry/sentry-elixir/blob/fba608c72a403cb0b21e9ebd8cd2cd313a1e3ecb/lib/sentry/envelope.ex). Without any server logs telling us what's wrong this is pretty hard to debug. If you report check-ins manually (with `Sentry.capture_check_in/1`, does it work?

**Comment by rodolfoBee** [comment]
@whatyouhide thank you for the info.
@gaprl from the crons team is also looking into the backend logs

**Comment by sl0thentr0py** [comment]
I don't think this is an SDK problem but we can wait for more server investigation before closing.

**Comment by sl0thentr0py** [comment]
@rodolfoBee as I said, rate limits and the `Dropped (Server)` reports are purely server side, I don't think it has to do with the SDK.

**Comment by whatyouhide** [comment]
It might be worth closing this particular issue so as to not confuse users into thinking it's an issue with the Elixir SDK?

**Comment by rodolfoBee** [comment]
Can it be transferred to the getsentry/sentry repo instead, so we can assign to the Crons team?

**Comment by whatyouhide** [comment]
@rodolfoBee even better yes, but I don't seem to have permissions to do that. @sl0thentr0py?
